lib.sh

```diff
@@ -291,7 +291,10 @@ input_box_flow() {
     while :
     do
         RESULT=$(input_box "$1" "$2")
-        if ! yesno_box_yes "Is this correct? $RESULT" "$2"
+        if [ -z "$RESULT" ]
+        then
+            msg_box "Input is empty, please try again."
+        elif ! yesno_box_yes "Is this correct? $RESULT" "$2"
```

Comment:
I am against this change because we won't be able to do something like this:
https://github.com/nextcloud/vm/pull/1449/files#diff-cb545bead2637c389f51b1ed9215d91eL61

Reply:
I think we should use the default `input_box` in that case, or don't ask the user to leave it empty.

Reply:
Regarding the bitwarden_mailconfig, maybe it would be better to grep that value and use it like we do in static IP?

https://github.com/nextcloud/vm/blob/master/network/static_ip.sh#L129

Reply:
I guess we have to rewrite the bitwarden-mailconfig script then in a follow-up.

Reply:
I opened #1453 for this